pkgs/servers/sql/mariadb/default.nix

```diff
@@ -1,7 +1,9 @@
 { stdenv, fetchurl, fetchFromGitHub, cmake, pkgconfig, makeWrapper, ncurses, zlib, xz, lzo, lz4, bzip2, snappy
 , libiconv, openssl, pcre, boost, judy, bison, libxml2, libkrb5, linux-pam, curl
-, libaio, libevent, jemalloc, cracklib, systemd, numactl, perl
+, libaio, libevent, jemalloc, cracklib, systemd, perl
 , fixDarwinDylibNames, cctools, CoreServices, less
+, withNUMA ? true, numactl
```

Comment:
I would say that numa is common enough in the area where database servers are used that we don't even need to add an option for it. Just drop the `withNUMA` option until someone finds a good reason for not having it.

Reply:
Fixed